pyfar/dsp/filter/fractional_octaves.py

```diff
@@ -329,6 +371,90 @@ def _coefficients_fractional_octave_bands(
     return sos
 
 
+def check_fractional_octave_band_filter_tolerance(
+            fractional_octave_band_filter, num_fractions, frequency_range,
+            tolerance_class):
+    """
+    Check magnitude tolerance of fractional octave band filters.
+
+    The tolerance is defined in [#]_ and returned by
+    :py:func:`~pyfar.constants.fractional_octave_filter_tolerance`.
+    See :py:func:`~pyfar.dsp.filter.fractional_octave_bands` for an example.
+
+    Parameters
+    ----------
+    fractional_octave_band_filter : FilterSOS
+        The fractional-octave-band filters.
+    num_fractions : int
+        The number of bands an octave is divided into. Must be ``1`` (octave
+        bands) or ``3`` (one-third octave bands).
+    frequency_range : array, tuple
```

Comment:
arnt this the cutoff_freqeuncies of the filter?

Reply:
I would wote for replacing ``frequency_range`` by the ``exact_center_frequencies``, this allows for flexibility and also making it easier to maintain. One reason is that ``frequency_range`` is just used to calcualte the ``exact_center_frequencies``, so there is additional overhead in checking consitancy.
in this case it would also allow it just e.g. 500 Hz octave and 2kHz Frequency, which wouldn't be possible right now.
it would also work for me to merge this PR and open a new issue for this.